README.md

```diff
@@ -22,8 +70,6 @@ npm run dev
 
 This spins up an application running at `http://localhost:8080/`. Your latest changes will be reflected here.
 
-![react-search development page](readmeImages/devEnv.png)
```

Comment:
I took this out because I'd prefer the reader's focus to be on the component from a usage POV. This will also make maintenance easier if we iterate on the dev page.